src/ir/objc.rs

```diff
@@ -131,14 +143,19 @@ impl ObjCInterface {
                     }
 
                 }
-                CXCursor_ObjCInstanceMethodDecl => {
+                CXCursor_ObjCInstanceMethodDecl |
+                CXCursor_ObjCClassMethodDecl => {
                     let name = c.spelling();
                     let signature =
                         FunctionSig::from_ty(&c.cur_type(), &c, ctx)
                             .expect("Invalid function sig");
-                    let method = ObjCInstanceMethod::new(&name, signature);
-
-                    interface.methods.push(method);
+                    if c.kind() == CXCursor_ObjCInstanceMethodDecl {
```

Comment:
nit: I think this may be cleaner with something like:

```rust
let is_class_method = c.kind() == CXCursor_ObjCInstanceMethodDecl;'
let method = ObjCMethod:new(&name, signature, is_class_method);
interface.add_method(method)
```

where `add_method` checks the `is_class_method` flag and pushes it to the right list?